change/@microsoft-teams-js-59318f7e-dc90-4d48-a1b2-5616fb3a6cef.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Fixed outdated references to 'core.' namespace in documentation",
```

Comment:
Preference is to enclose namespaces or other API types/interfaces/etc in backtick quotes.